src/code/entities/bank3.asm

```diff
@@ -2823,7 +2823,7 @@ HeartContainerEntityHandler::
     ; Finished setting status bits for rooms, delete this
     jp   UnloadEntityAndReturn
 
-HoldEntityAboveLink::
+HoldEntityskipSwimUp::
```

Comment:
Was this renaming intended? I can't really figure out what the function does with the new naming.

Reply:
It wasn't. It leaked from a rename of a label in the Blooper file. But it's fixed now.